packages/browser-destinations/src/destinations/upollo/identifyUser/index.ts

```diff
@@ -70,7 +70,7 @@ const identifyUser: BrowserActionDefinition<Settings, UpolloClient, Payload> = {
       customerSuppliedValues: payload.custom_traits ? toCustomValues(payload.custom_traits) : undefined
     }
 
-    void UpClient.track(userInfo)
+    return UpClient.track(userInfo)
```

Comment:
Does `perform` expect us to return something, and what format?
I would have thought we dont want to return anything.

```suggestion
    void UpClient.track(userInfo)
```